### DISCUSSION TITLE
useTransition timeout independently

### DISCUSSION THREAD

**Discussion by emjaksa** [pmndrs/react-spring#1076] [Support]
I am trying to do something very similar to the notifications hub [example](https://codesandbox.io/embed/7mqy09jyq) but the issue I am facing is that when you add a notification and then add a second one, the first notification slows down and they both timeout/fade out at the same time. How would I make them timeout out independently?

**Comment by aleclarson** [comment]
There's a fork of that demo for v9 (see here: https://github.com/react-spring/react-spring-examples/blob/v9/demos/hooks/notification-hub/index.js), but no sandbox for it atm.
